bodhi/server/mail.py

```diff
@@ -281,10 +281,40 @@
 
 --------------------------------------------------------------------------------
 %(notes)s%(changelog)s%(references)s
+This update can be installed with the "dnf" update programs.  Use
+su -c 'dnf%(yum_repository)s update %(name)s' at the command line.
+For more information, refer to "Package Management",
+available at https://docs.fedoraproject.org
+
+All packages are signed with the Fedora EPEL GPG key. More details on the
+GPG keys used by the Fedora Project can be found at
+https://fedoraproject.org/keys
+--------------------------------------------------------------------------------
+"""
+
+# message template for legacy systems, still pointing users to use "yum"
+fedora_epel_legacy_errata_template = u"""\
```

Comment:
I don't think we need to add a "legacy" template just yet, since nothing is currently using the ```fedora_epel_errata_template``` defined above. I assume this was to think ahead for EL8, but I think we can just cross that bridge when we get to it. For now, let's just keep less text around and use the already existing template name.